deploy/06_asset/10_deploy_genesis_bouncer.ts

```diff
@@ -0,0 +1,39 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts} = hre;
+  const {deploy, read, execute} = deployments;
+
+  const {
+    genesisBouncerAdmin,
+    genesisMinter,
+    deployer,
+  } = await getNamedAccounts();
+
+  const Asset = await deployments.get('Asset');
+
+  //(ERC1155ERC721 asset, address genesisAdmin, address firstMinter
```

Comment:
Not quite sure what this comment is saying?

Reply:
oh, that was is the type the constructor argument (copied from the contract itself

Reply:
Ok - I wondered if there was a bit missing because only 1 bracket. This is fine!